cli/Squidex.CLI/Squidex.CLI.Core/Commands/Implementation/Sync/Assets/AssetsSynchronizer.cs

```diff
@@ -63,21 +63,24 @@ async Task SaveAsync()
 
             await session.Client.Assets.GetAllAsync(async asset =>
             {
-                var model = asset.ToModel();
+                if (asset.Created >= options.MaxAgeDate || asset.LastModified >= options.MaxAgeDate)
```

Comment:
LastModified should always be the same as created for new assets and contents so the condition can be simplified. I also tend to invert the "if" in this case so reduce nesting.